setup.py

```diff
@@ -58,7 +58,6 @@ def write_version_file():
     pytorch_dep += "==" + os.getenv("PYTORCH_VERSION")
 
 requirements = [
-    "typing_extensions",
```

Comment:
We only ever used this for `Literal`, which is available through `typing` in the standard library for Python >= 3.8.